### PR TITLE
Add delete feature modal

### PR DESCRIPTION
**Pull request by m99coder**
Issue: https://github.com/zalando-incubator/pazuzu-ui/issues/71
* 92c4afe - added notice if there are no features to list view
* ebac189 - fixed form label animations and search bar
* 5a2b319 - added materialize modal to confirm feature delete
* 21c27d5 - added angular2-materialize module, added some tooltips and wave effects
* c8dd575 - added simple delete trigger button for demo

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -71,15 +71,23 @@ main .editor * {
...
+  color: #ff5722;
 }
 
+/*
 .textarea-field label {
   font-size: 0.8rem;
   -webkit-transform: translateY(-140%);
   transform: translateY(-140%);
 }
+*/
```
Comment (netme) [inline src/styles.css]: Maybe we should already remove it?

File: src/app/feature-list/feature-list.component.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ export class FeatureListComponent implements OnInit, OnDestroy {
     this.page = Math.ceil((+this.params.get('offset') + 1) / +this.params.get('limit'));
 
     // number of pages
-    this.pages = Math.ceil(this.total / +this.params.get('limit'));
+    this.pages = Math.max(1, Math.ceil(this.total / +this.params.get('limit')));
```
Comment (netme) [inline src/app/feature-list/feature-list.component.ts]: Nice catch!

**Comment by netme** [comment]
Please also point this PR to a corresponding issue on GitHub

**Comment by netme** [comment]
👍

**Comment by falmp** [comment]
Looks good to me too. 👍

**Comment by falmp** [comment]
👍

**Pull request merged**